doc/sphinx/proof-engine/tactics.rst

```diff
@@ -215,16 +221,13 @@ Applying theorems
          | Ok : bool -> Option.
 
          Definition get : forall x:Option, x <> Fail -> bool.
-
-         refine
-           (fun x:Option =>
-           match x return x <> Fail -> bool with
-           | Fail => _
-           | Ok b => fun _ => b
-           end).
-
-         intros; absurd (Fail = Fail); trivial.
-
+           refine
```

Comment:
Do we want to add `Proof.` here?

Reply:
I don't know. People generally use `Proof.` with `Qed.` but with `Defined.`? I have no idea.